### ISSUE TITLE
Are you planning to support tensorboard in eager mode?

### ISSUE THREAD

**Issue by so77id** [wookayin/tensorflow-plot#8]

**Comment by wookayin** [comment]
Yes, I plan to support it. But in eager mode you don't need to bother use a plot operation -- just you can create a summary and add it to summarywriter. I think you can still do it even using the current one, but I also feel we need more handy API and direct directives for eager mode. Stay tuned!

**Comment by wookayin** [comment]
In eager mode, the plot operation can be called multiple times; with the current implementation duplicate instances of python function object will be created again and again. We may need a clever mechanism of caching.
